packages/neutrino-middleware-hot/index.js

```diff
@@ -1,3 +1,10 @@
-const { HotModuleReplacementPlugin } = require('webpack');
+const { HotModuleReplacementPlugin, NamedModulesPlugin } = require('webpack');
 
-module.exports = ({ config }) => config.plugin('hot').use(HotModuleReplacementPlugin);
+module.exports = ({ config }) => {
+  config
+    .plugin('hot')
+      .use(HotModuleReplacementPlugin)
+      .end()
+    .plugin('named-modules')
```

Comment:
It's not clear whether the order of the plugins matters here - for that and the fact that the suggestion of using these plugins together isn't actually documented anywhere (other then the console output), I've filed webpack/webpack.js.org/issues/1623.